### ISSUE TITLE
stoi loss The output value is negative

### ISSUE THREAD

**Issue by LTuohaha** [mpariente/pytorch_stoi#11]
When I used torch-stoi as loss function for my network , the output of stoi is negative all time, does it normal or just I used it wrong.
My usage just like the code belowk, I put **enhanced speech** as **first** input parameter while the _**clean speech**_ as _**second**_.

Please Please correct me and tell me the right way!!
```
noisy_speech = torch.randn(2, 16000)
clean_speech = torch.randn(2, 16000)
# Estimate clean speech
est_speech = nnet(noisy_speech)
# Compute loss and backward (then step etc...)
loss_batch = loss_func(est_speech, clean_speech)
loss_batch.mean().backward()
```

**Comment by mpariente** [comment]
Yes it's normal. 
For STOI, the higher, the better. 
Gradient descent minimizes loss functions. 
So we optimize over `- stoi`. This is the same fo SDR for example.
